package day45_Exceptions;

public class ThrowKeyword2 {
    public static void main(String[] args) {
        //ArithmeticException obj1 = new ArithmeticException(); //u cannot use any data after throw
        //throw obj1;

        new ArithmeticException();
        throw new ArithmeticException();

    }



}
